demos/sql-injection-attack-prevention/azuredeploy.parameters.json

```diff
@@ -7,6 +7,12 @@
     },
     "sqlServerPassword": {
       "value": "GEN-PASSWORD"
+    },
+    "bacpacUriSasToken": {
+        "value": "GEN-PASSWORD"
+    },
+    "bacpacUri": {
+        "value": "https://azbotstorage.blob.core.windows.net/bacpacs/contosoclinic.bacpac"
```

Comment:
use GEN-BACPAC-URI - and then remove the us gov param file since they will be the same.

Reply:
Changes done